Update User Guide (#279)
AY1920S1-CS2103-T16-3/main: docs/UserGuide.adoc

.  Copy the file to the folder you want to use as the home folder for your Weme.
.  Double-click the file to start the app. The GUI should appear in a few seconds.
+

> image::../Ui.png[width="790"]

Reviewer: Can't see anything.

> image::../Ui.png[width="790"]

Reviewer 2: Wrong path

Reviewer 2: Remove the leading `../`, please test before push

* *import*
* *preferences*

Example:

> * Switch to `templates` tab using `tab templates` command:

Reviewer 2: ```suggestion
* `tab templates` +
 Switches to the `templates` tab.
```

Please ensure consistency with the rest of the user guide.

* `add p//Users/Me/Downloads/dgirl_oof.jpg t/Funny` +
Adds a meme with the image from `/Users/Me/Downloads/dgirl_oof.jpg` and tag `Funny`

> Example of adding a meme with description and tags: +

Reviewer 2: ```suggestion
Illustration of adding a meme at `pathToMeme` with description `Cat and child` and tags `baby` and `cuteCat`: +
```

`add p/pathToMeme d/Cat and child t/baby t/cuteCate`

Before command:

image::MemeAddCommand2.png[width="790"]
After command:

image::MemeAddCommand.png[width="790"]

> 

Reviewer 2: Put these after the box explaining relative and absolute paths.

* `edit 2 d/Random Meme t/` +
Edits the description of the 2nd meme to be `Random Meme` and clears all existing tags.

> Example of editing a meme's description: +

Reviewer 2: ```suggestion
Illustration of editing the description of the meme at index 1 in the displayed list: +
```

* `find pikachu` +
Returns memes whose tags contain `pikachu`

> Example of finding memes with a tag +

Reviewer 2: ```suggestion
Illustration of finding memes with tag `cute` +
```

`find cute`

> After command:

Reviewer 2: Missing before command screenshot
